newIDE/app/src/InstancesEditor/InstancesResizer.js

```diff
@@ -1,4 +1,6 @@
 // @flow
+import Rectangle from '../Utils/Rectangle';
+
 type Dimension = {|
```

Comment:
I think the Dimension type can be removed?

Reply:
Yes, I removed it.